### DISCUSSION TITLE
Allow sorting on *ALL* columns

### DISCUSSION THREAD

**Discussion by JesseWeinstein** [tosdr/edit.tosdr.org#994] [Ideas]
Rather than having explicit buttons that only allow sorting on a few columns -- can we just implement a general sortable-columns library, that allows for arbitrary sorting on all the columns? Specifically, I'd love to be able to sort on whether or not documents are available.

**Comment by madoleary** [comment]
This is a good idea. I think there's a tool in Rails that we could use to help with this. I'll look into it once I've gotten a couple of other things off my to-do list.
